pkg/apis/openshiftcontrollermanager/v1alpha1/types.go

```diff
@@ -39,10 +39,24 @@ type OpenShiftControllerManagerOperatorConfigSpec struct {
 	// observedConfig holds a sparse config that controller has observed from the cluster state.  It exists in spec because
 	// it causes action for the operator
 	ObservedConfig runtime.RawExtension `json:"observedConfig"`
+
+	// additionalTrustedCA references the additional trusted certificate authorities that operator should attempt to configure for the build controller.
+	AdditionalTrustedCA *AdditionalTrustedCA `json:"additionalTrustedCA,omitempty"`
+}
+
+type AdditionalTrustedCA struct {
+	// sha1Hash contains the sha1 hash of the CA bundle data
+	SHA1Hash string `json:"sha1Hash,omitempty" protobuf:"bytes,1,opt,name=sha1Hash"`
+
+	// configMapName is the name of the ConfigMap in the openshift-config namespace containing the additional trusted CAs for the build controller.
+	ConfigMapName string `json:"configMap,omitempty" protobuf:"bytes,2,opt,name=configMap"`
```

Comment:
CA data stores the SHA1Hash of the merged CA bundle, and the name of the ConfigMap.
Per discussion with @bparees and @deads2k, cluster admin provided configurations will live in the `openshift-config` namespace.

Reply:
SHA probably belongs only in status, not spec, as it is not something the user gets to set.

Reply:
I did this for consistency, though we can probably get away without it.

Reply:
if we don't need it, let's get rid of it.  just a source of confusion otherwise.

Reply:
Now I remember...I do this so we can capture that the cluster admin updated the data in the referenced ConfigMap. I want to avoid races due to having an informer for `openshift-config` here and in the main operator controller.